packages/neocmakelsp/package.yaml

```diff
@@ -10,7 +10,16 @@ categories:
   - LSP
 
 source:
-  id: pkg:cargo/neocmakelsp@0.7.3
+  id: pkg:github/Decodetalkers/neocmakelsp@v0.7.3
+  asset:
+    - target: darwin_arm64
+      file: neocmakelsp-aarch64-apple-darwin
+    - target: darwin_x64
+      file: neocmakelsp-x86_64-apple-darwin
+    - target: linux_x64_gnu
+      file: neocmakelsp-x86_64-unknown-linux-gnu
```

Comment:
We probably want to target the `-musl` variant for broader support (can remove the `_gnu` specifier then).

Reply:
Upstream has both gnu and musl, I updated the PR to support both